### ISSUE TITLE
Print the output of postinstall scripts

### ISSUE THREAD

**Issue by hugo-t-b** [pnpm/pnpm#4650]
- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/pnpm/pnpm/blob/main/CONTRIBUTING.md))

**Describe the user story**

Many npm packages (e.g. Cypress) have `postinstall` scripts which can take a number of minutes to run. Normally, these scripts log their progress, which helps users to know how long is left and whether the script is still running. However, pnpm does not show the output of `postinstall` scripts.

**Describe the solution you'd like**

Ideally, pnpm would show the output of `postinstall` scripts.

**Describe the drawbacks of your solution**

This could make the output of commands like `pnpm install` long and cluttered.

**Describe alternatives you've considered**

It would be possible to log only the important parts of the output of `postinstall` scripts (e.g. progress bars), but this would be very difficult to implement.

**Comment by zkochan** [comment]
Now that we use side-effects cache by default, it might be OK to print all that output.

Also, you can use `--reporter=append-only` to see the that output.

**Comment by hugo-t-b** [comment]
That is helpful, thank you. Is there a way to set `--reporter=append-only` as the default?

**Comment by zkochan** [comment]
Yes, you can add it to a `.npmrc` file as `reporter=append-only`.

Or you may run `pnpm config set reporter append-only` at will be set globally.

**Comment by Waujito** [comment]
> Now that we use side-effects cache by default, it might be OK to print all that output.
> 
> Also, you can use `--reporter=append-only` to see the that output.

Hello. I think, this is not clean way...
![изображение](https://user-images.githubusercontent.com/53368344/182838636-8d9a1766-f68c-4ea1-b967-74ef9249ef64.png)
But this is better instead of it
![изображение](https://user-images.githubusercontent.com/53368344/182838776-44f0b041-16fc-463c-b179-f9affc24c4a7.png)
